docs/development/quickstart.md

```diff
@@ -2,7 +2,7 @@
 
 # Development Quickstart
 
-Test Pilot uses Node.js [v6.x LTS](https://nodejs.org/dist/latest-v6.x/) for development. You may be able to get by using
+Test Pilot uses Node.js v6.2.0 for development. You may be able to get by using
```

Comment:
Danny updated this file yesterday in ece1ba14c1292c4f9763c7f8b08e7006e3e329d7; you may need to rebase against an updated master.